ivy_tests/test_ivy/test_frontends/test_numpy/test_random/test_functions.py

```diff
@@ -235,3 +235,30 @@ def test_numpy_multinomial(
         pvals=np.array([1 / n] * n, dtype=dtype[0]),
         size=size,
     )
+
+
+# permutation
+@handle_frontend_test(
+    fn_tree="numpy.random.permutation",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1
+    ),
+)
+def test_numpy_permutation(
+    dtype_and_x,
+    frontend,
+    test_flags,
```

Comment:
I can replicate the failure that you are facing, however, I do get an error which I believe it's on our side. I'll ask about it and update soon!